.circleci/workspace_status.sh

```diff
@@ -25,3 +28,15 @@ echo "GIT_SHA ${git_sha}"
 # Tag name, or GIT_SHA if not on a tag
 git_ref=$(git describe --tags --no-match --always --abbrev=40 --dirty | sed -E 's/^.*-g([0-9a-f]{40}-?.*)$/\1/')
 echo "GIT_REF ${git_ref}"
+
+# Plain git revision for linkstamping.
+echo "BUILD_SCM_REVISION" $(git rev-parse HEAD)
```

Comment:
Since this is just a string, should we consider the more advanced form from `GIT_REF`?

This is just the Bazel default.